build-tools/packages/build-tools/src/common/npmPackage.ts

```diff
@@ -202,6 +202,18 @@ export class Package {
 		return path.dirname(this.packageJsonFileName);
 	}
 
+	public getLockFilePath() {
```

Comment:
Docs. Is it a file path or just a directory?

Reply:
full path. Added comment